.github/workflows/test.yml

```
@@ -72,21 +72,25 @@ jobs:
         shell: bash
         run: ci/scripts/build_example.sh $(pwd)/example
   windows:
-    name: AMD64 Windows 2019
-    runs-on: windows-2019
+    name: AMD64 Windows 2022
+    runs-on: windows-2022
     timeout-minutes: 30
     steps:
       - name: Checkout iceberg-cpp
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
+      - name: Install ZLIB
+        shell: cmd
+        run: |
+          vcpkg install zlib:x64-windows
       - name: Build Iceberg
         shell: cmd
         run: |
-          call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
+          call "C:\Program Files (x86)\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
           bash -c "ci/scripts/build_iceberg.sh $(pwd)
```

It seems that '"' isn't closed:

```suggestion
          bash -c "ci/scripts/build_iceberg.sh $(pwd)"
```